src/components/accordion/index.md.njk

```diff
@@ -17,83 +17,119 @@ The accordion component lets users show and hide sections of related content on
 
 ## When to use this component
 
-Only use an accordion if there's evidence it’s helpful for users to:
+Only use an accordion if there's evidence it’s helpful for the user to:
 
 - see an overview of multiple, related sections of content
-- show and hide those sections as needed
+- choose to show and hide sections that are relevant to them
+- look across information that might otherwise be on different pages
 
-Accordions can work well for people who use a service regularly, for example users of caseworking systems who need to perform familiar tasks quickly.
+For example, an accordion can work well if the user needs to reveal and compare information that’s relevant to them.
+
+Accordions can also work well for people who use a service regularly. For example, users of caseworking systems who need to do familiar tasks quickly.
+
+Test with users to decide if using an accordion outweighs the potential problems with hiding content.
 
 ## When not to use this component
 
-Accordions hide content from users and not everyone will notice them or understand how they work. For this reason do not use an accordion for content which is essential to all users.
+Accordions hide content from the user. Not all users will notice them or understand how they work. For this reason, you should only use them in specific situations and if user research supports it. 
+
+Do not use an accordion for content that all users need to see.
 
-Test your content without an accordion first. Consider if it’s better to:
+Test your content without an accordion first. Well-written and structured content, as shown in the [Content design: writing for GOV.UK](https://www.gov.uk/guidance/content-design/writing-for-gov-uk) guidance, can remove the need to use an accordion. 
+
+It’s usually better to:
 
 - simplify and reduce the amount of content
 - split the content across multiple pages
 - keep the content on a single page, separated by headings
-- use a list of links to let users navigate quickly to specific sections of content
-
-Do not use the accordion component if the amount of content it would need to contain will make the page slow to load.
+- use a list of links at the start of the page (known as 'anchor links') to take the user to particular sections of a page
 
-Accordions work best for simple content and links. Do not use accordions to split up a series of questions into sections. Use separate pages instead.
+Accordions work best for simple content and links. Do not use accordions to split up a series of questions. Use separate pages instead.
 
-## Accordions, tabs and details
+Avoid putting accordions within accordions, as it will make content difficult to find.
```

Comment:
Thanks for including this in @calvin-lau-sig7 👍 

I would be tempted to make this stronger by recommending not using them at all in this way as the component was not designed to be nested, we haven't tested them in this way with assistive technologies, and we are not aware of any user research done with nested accordions. @christopherthomasdesign might be able to add something more about why nested accordions aren't great from usability point of view.

Reply:
Yeah I was gonna say the same thing @hannalaakso – I'd say 'do not'. On the most basic level, it really just isn't a simple enough design in any context, it would look a mess and very likely confuse people. It also seems like something that would be really difficult for screen reader users to navigate, though I haven't tested that.

Reply:
+1 on this

Reply:
```suggestion
Do not put accordions within accordions, as it will make content difficult to find.
```